browser-test/src/application_navigation.test.ts

```diff
@@ -269,6 +269,31 @@ describe('Applicant navigation flow', () => {
       await validateAccessibility(page)
       await validateScreenshot(page, 'program-submission')
     })
+
+    it('shows error with incomplete submission', async () => {
+      const {page, applicantQuestions} = ctx
+      await loginAsGuest(page)
+      await selectApplicantLanguage(page, 'English')
+      await applicantQuestions.clickApplyProgramButton(programName)
```

Comment:
At this point, hasn't the user never applied to the program (since in the beforeAll the admin has just created questions) and they click into the application and they see the summary of what's to come then they click continue to proceed to the first question. I don't see where there's been an update to the application since the user started filling it out in this flow? So confused why they'd see the error at this point? Could be missing something though.

Reply:
The setup for this issue isn't doable in our test framework as it would require 2 browser contexts.

So per the comment below, I'm manipulating the page to let us submit an incomplete application by changing the existing button in the page to do a submit rather than an edit action.  That submit will then result in submitting an incomplete application and the banner.

Reply:
Ohhh okay that makes sense. Maybe add "The UI correctly won't let us submit, since responses haven't been filled out" or something to the comment below?

Reply:
done